src/common/stream.c

```diff
@@ -168,7 +145,9 @@ struct stream {
 	union {
 		void *p;
 		int i;
+#if !defined(HAVE_EMBEDDED_JAVA) || !defined(NATIVE_WIN32)
 		SOCKET s;
```

Comment:
WHat do you need the SOCKET for? We can remove it...

Reply:
Nothing! I was getting a compilation error on Windows. We can try to remove most of the content of the stream.c module.

Reply:
Fixed (removed it).